src/core/cpu_patches.cpp

```diff
@@ -746,17 +950,19 @@ static void TryPatchAot(void* code_address, u64 code_size) {
     }
 }
 
-static bool PatchesAccessViolationHandler(void* code_address, void* fault_address, bool is_write) {
-    return TryPatchJit(code_address);
+static bool PatchesAccessViolationHandler(void* context, void* /* fault_address */) {
+    return TryPatchJit(context);
```

Comment:
```suggestion
    return TryPatchJit(Common::GetRip(context));
```

Reply:
Yikes, 5:16 AM coding session not going great